src/sys/ptrace.rs

```diff
@@ -140,3 +140,51 @@ pub fn setsiginfo(pid: Pid, sig: &siginfo_t) -> Result<()> {
         Err(e) => Err(e),
     }
 }
+
+/// Sets the process as traceable with `PTRACE_TRACEME`
+pub fn traceme() -> Result<()> {
+    unsafe {
+        ptrace(
+            ptrace::PTRACE_TRACEME,
+            Pid::from_raw(0),
+            ptr::null_mut(),
+            ptr::null_mut(),
+        ).map(|_| ()) // ignore the useless return value
+    }
+}
+
+/// Makes the `PTRACE_SYSCALL` request to ptrace
```

Comment:
Should I copy-paste the ptrace manual? I thought that the user should be familiar with what ptrace does.